README.md

```diff
@@ -88,6 +88,7 @@ in `app/build.gradle`:
 dexcount {
     format = "list"
     includeClasses = false
+    includeClassCount = true
```

Comment:
Nit: this should be `false` to match the default value in `DexMethodCountExtension`